imports/api/README.md

```diff
@@ -0,0 +1,20 @@
+Sarif File Changes for 
```

Comment:
Could you also add to the readme a small user manual about how to run the application?
It used to be "run the command 'meteor'", but now to set up the database, we have to run something extra, right?
Could you make that explicit here? Thanks.